restart_puma.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+# simple script to restart puma
+# e.g. docker exec -it 'cellect_container_name' bash -c "/cellect_panoptes/restart_puma.sh"
+kill -s SIGUSR2 `ps aux | grep [p]uma | awk '{print $2}'`
```

Comment:
Not the way we've got it running but we could do: 
https://github.com/puma/puma/blob/master/examples/config.rb